p2p/src/main/java/bisq/network/p2p/peers/PeerManager.java

```diff
@@ -505,59 +632,62 @@ private void printReportedPeers() {
     }
 
     private void printNewReportedPeers(Set<Peer> reportedPeers) {
-        //noinspection ConstantConditions
         if (PRINT_REPORTED_PEERS_DETAILS) {
             StringBuilder result = new StringBuilder("We received new reportedPeers:");
             List<Peer> reportedPeersClone = new ArrayList<>(reportedPeers);
-            reportedPeersClone.stream().forEach(e -> result.append("\n\t").append(e));
+            reportedPeersClone.forEach(e -> result.append("\n\t").append(e));
             log.trace(result.toString());
         }
         log.debug("Number of new arrived reported peers: {}", reportedPeers.size());
     }
 
 
     ///////////////////////////////////////////////////////////////////////////////////////////
-    //  Persisted list
+    //  Persisted peers
     ///////////////////////////////////////////////////////////////////////////////////////////
 
     private boolean removePersistedPeer(Peer persistedPeer) {
-        if (persistedPeers.contains(persistedPeer)) {
-            persistedPeers.remove(persistedPeer);
-            peerList.setAll(persistedPeers);
-            persistenceManager.requestPersistence();
+        if (getPersistedPeers().contains(persistedPeer)) {
+            //getPersistedPeers().remove(persistedPeer);
```

Comment:
I don't understand why this line is commented out. Seems the peer is not removed as it should now.

Reply:
Ups that was for dev testing as in regtest environment the failed connection attempts quickly rise and then the peers get removed... will enable it again.